data_steward/analytics/cdr_ops/controlled_tier_qc/check_controlled_tier_part2.py

```diff
@@ -1157,6 +1157,67 @@ def my_sql(table_name, column_name):
         },
         ignore_index=True)
 
+# # Query 16:  All the data from drug_era, condition_era, and dose_era tables is dropped
+
+# +
+era_tables = ['dose_era', 'drug_era', 'condition_era']
+
+
+def my_query(table_era):
+
+    query = JINJA_ENV.from_string("""
+      WITH df1 AS (
+        SELECT 
+          person_id
```

Comment:
@lledezma suggestion, instead of using person_id in the query, you can use {{table_era}}_id. Since we have a primary_key for these tables it is better to use those keys instead of person_id.